contracts/test/Gateway.t.sol

```diff
@@ -903,10 +922,27 @@ contract GatewayTest is Test {
         // register token first
         uint256 fee = IGateway(address(gateway)).quoteRegisterTokenFee();
         IGateway(address(gateway)).registerToken{value: fee}(address(token));
-
         fee = IGateway(address(gateway)).quoteSendTokenFee(address(token), destPara, 0);
 
         vm.expectRevert(Assets.InvalidDestinationFee.selector);
         IGateway(address(gateway)).sendToken{value: fee}(address(token), destPara, recipientAddress32, 0, 1);
     }
+
+    function testSendTokenWithLargeFeeToForeignDestWithInvalidFee() public {
```

Comment:
Same here, maybe `testSendTokenWithLargeInvalidFeeoForeignDest`.

Reply:
Addressed in [23fe816](https://github.com/Snowfork/snowbridge/pull/1168/commits/23fe816ce2d5ac3e1242e87a27f816571dcee77e)

Reply:
Lol I see I made a typo here, sorry! 🙈 Should be `testSendTokenWithLargeInvalidFeeToForeignDest`.